### PR TITLE
Add notification for season 3 incentives

### PR DESCRIPTION
**Pull request by rosepuppy**
Screenshot 2024-02-21 at 3 18 48 PM

---

## Constants/Types

* `src/constants/analytics.ts`
  * Add `AnalyticsEvent.NotificationAction`

## Hooks

* `src/hooks/useNotificationTypes.tsx`
  * add support for `NotificationType.ChaosLabs`
  * Set expiration date to be 2024-03-08
  * trigger the notification. Hardcode the values
  * Navigate to token page on click

* `src/hooks/useNotifications.tsx`
  * Add `NotificationType.ChaosLabs`
  * Add tracking for whenever notifications are clicked to `onNotificationAction`

## Packages

* Update v4-localization

### CODE REVIEW

File: src/hooks/useNotificationTypes.tsx [hunk 1]
```diff
@@ -280,6 +280,43 @@ export const notificationTypes: NotificationTypeConfig[] = [
...
+    useTrigger: ({ trigger }) => {
+      const stringGetter = useStringGetter();
+      const expirationDate = new Date('2024-03-08T23:59:59');
+      const currentDate = new Date();
+      useEffect(() => {
+        if (currentDate > expirationDate) return;
+        trigger(
+          'chaos-labs-s3',
+          {
+            icon: <AssetIcon symbol="DYDX" />,
```
Comment (jaredvu) [inline src/hooks/useNotificationTypes.tsx:294]: should we use the `chainTokenLabel` from `useTokenConfig` here?

File: src/hooks/useNotifications.tsx [hunk 2]
```diff
@@ -13,6 +14,7 @@ import {
 import { useLocalStorage } from './useLocalStorage';
 import { notificationTypes } from './useNotificationTypes';
 
+import { track } from '@/lib/analytics';
 import { renderSvgToDataUrl } from '../lib/renderSvgToDataUrl';
```
Comment (jaredvu) [inline src/hooks/useNotifications.tsx:18]: nit: use `@lib/` for `renderToSvgDataUrl` and import before relative imports

File: src/hooks/useNotifications.tsx [hunk 3]
```diff
@@ -82,6 +84,7 @@ const useNotificationsContext = () => {
         [NotificationType.AbacusGenerated]: true,
         [NotificationType.SquidTransfer]: true,
         [NotificationType.ReleaseUpdates]: true,
+        [NotificationType.ChaosLabs]: true,
```
Comment (jaredvu) [inline src/hooks/useNotifications.tsx:87]: This `NotificationType.ChaosLabs` should probably be bucketed w/ `ReleaseUpdates` or another `NotificationType`. Check out `PreferencesDialog` to see how these NotificationTypes are being used.

Comment (rosepuppy) [reply src/hooks/useNotifications.tsx:87]: good call 👍

**Review by jaredvu — APPROVED** [review]

**Pull request merged**